src/Xamarin.Android.Build.Tasks/Tasks/Lint.cs

```diff
@@ -49,6 +51,9 @@ public class Lint : ToolTask
 		[Required]
 		public string IntermediateOutputPath { get; set; }
 
+		[Required]
+		public string LintToolVersion { get; set; }
```

Comment:
Does this need to be a *required* property when we now execute `lint --version` within the task? I think we can remove this property, no?

Reply:
The presence of `[Required]` *should* result in a build failure...